Assets/StreamingAssets/Text/ConversationText.txt

```diff
@@ -1,45 +1,45 @@
-- Text database for conversations used in talk window.

```

Comment:
Entire file is edited due to line break conversion, without any meaningful change. Can you amend commit to remove this file?

Reply:
I know, I initially changed something to this file then reverted it. However, original file has messed up line endings with a mix of both Windows and Unix eol so I let it here. I'll push this to a separate commit if you prefer.

Reply:
I always have problems with line endings warnings on my windows/visual studio systems. I installed git/tortoise git with all standard/recommended settings but I somehow end up with those warnings all the time. Who knows how to get rid of these messages/problems?

Reply:
I think the eol option has to be set to "check out as is commit Unix style "

Reply:
That's fine then, feel free to make a commit for it. thank you :)

Reply:
I removed the file for now :)

Reply:
> 
> 
> I think the eol option has to be set to "check out as is commit Unix style "

so the (tortoise) git default settings are wrong then if I understand correctly?

Reply:
Yes, since DFU files use Unix-style line endings, Git for Windows default settings ("checkout as-is, commit as-is") can lead to files having a mix of Windows and Unix line endings. This, because Visual Studio does not respect current file end of lines format when pasting code in it. However, changing the global Git option is usually not recommended, especially if you're working on several projects. This is the case for me, but my difftool (Beyond Compare, not free but by far the best one :)) clearly shows if line endings are messed up.

The best solution there is to force Visual Studio to use LF line endings with DFU. There is a plugin for that: https://marketplace.visualstudio.com/items?itemName=JakubBielawa.LineEndingsUnifier.

Though I didn't check it yet.